applications/FluidDynamicsApplication/tests/cpp_tests/test_alternative_qs_vms_dem_coupled_quadratic_element_3D.cpp

```diff
@@ -0,0 +1,138 @@
+//    |  /           |
```

Comment:
I'd add this test to current `test_alternative_qs_vms_dem_coupled_element.cpp` under the very same reasoning I posted above.

Reply:
I thought that it would be better to split these tests in three in order to ease the detection of errors. However, I can add them in only one.

Reply:
The tests are already split by the macro.

Reply:
Ahh... you mean to write them in the same .cpp file. I thought you meant to join them in only one test.

Reply:
Exactly this.